workers/unity/Packages/com.improbable.gdk.core/Systems/CommandBookkeepingSystem.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core.Commands;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    public class CommandBookkeepingSystem : ComponentSystem
```

Comment:
I think this name needs to be more specific. The book keeping is for a specific thing. Maybe CommandRequestTrackerSystem. That way we could also use it to see what currently pending commands we have. We could also use this to invalidate things when entities are removed etc.

Also bad to give it a generic name in case we eventually want to have some system that keeps track of what responses we have sent etc.

Reply:
Is there a reason we would want to split it out for Responses (in the future potentially)?